Managers/UInputMan.cpp

```diff
@@ -1012,9 +1012,8 @@ namespace RTE {
 		int mousePlayer = MouseUsedByPlayer();
 		// TODO: Figure out a less shit solution to updating the mouse in GUIs when there are no mouse players configured, i.e. no player input scheme is using mouse+keyboard. For not just check if we're out of Activity.
 		if (!g_ActivityMan.IsInActivity() || mousePlayer != Players::NoPlayer) {
-			// Multiplying by 30 for sensitivity. TODO: Make sensitivity slider 1-50;
-			m_AnalogMouseData.m_X += m_RawMouseMovement.m_X * 3;
-			m_AnalogMouseData.m_Y += m_RawMouseMovement.m_Y * 3;
+			m_AnalogMouseData.m_X += m_RawMouseMovement.m_X;
```

Comment:
Does this affect current sensitivity? Is this a good idea?

Reply:
OOOOh that's why mouse felt so slow suddenly. This made in by accident and I also forgot to update the default sensitivity value (available in settings ini). I've always wanted to get rid of that magic *3, I can always separate it back into it's own PR

Reply:
Cool. If you are happy with reverting this, maybe I can just throw in a sensitivity slider into the menu. I don't think it'd take long